offload/libomptarget/interface.cpp

```diff
@@ -354,17 +355,21 @@ static inline int targetKernel(ident_t *Loc, int64_t DeviceId, int32_t NumTeams,
     printKernelArguments(Loc, DeviceId, KernelArgs->NumArgs,
                          KernelArgs->ArgSizes, KernelArgs->ArgTypes,
                          KernelArgs->ArgNames, "Entering OpenMP kernel");
-#ifdef OMPTARGET_DEBUG
-  for (uint32_t I = 0; I < KernelArgs->NumArgs; ++I) {
-    DP("Entry %2d: Base=" DPxMOD ", Begin=" DPxMOD ", Size=%" PRId64
-       ", Type=0x%" PRIx64 ", Name=%s\n",
-       I, DPxPTR(KernelArgs->ArgBasePtrs[I]), DPxPTR(KernelArgs->ArgPtrs[I]),
-       KernelArgs->ArgSizes[I], KernelArgs->ArgTypes[I],
-       (KernelArgs->ArgNames)
-           ? getNameFromMapping(KernelArgs->ArgNames[I]).c_str()
-           : "unknown");
-  }
-#endif
+
+  ODBG_OS(ODT_Kernel, [&](llvm::raw_ostream &Os) {
+    for (uint32_t I = 0; I < KernelArgs->NumArgs; ++I) {
+      Os << "Entry " << llvm::format_decimal(I, 2)
+         << " Base=" << KernelArgs->ArgBasePtrs[I]
```

Comment:
Missing ":" before Base=. It's causing offloading/target_map_for_member_data.cpp to fail.